spec.bs

```diff
@@ -3907,8 +3981,13 @@ from querying the server during an auction.
     1. [=list/For each=] |igAd| of |ig|'s [=interest group/ads=]:
       1. Compute the |adHashCode| following [=compute the key hash of ad=] for |ig| and |igAd|.
       1. Run [=update k-anonymity cache for key=] on |adHashCode|.
-      1. Compute the |adReportingHashCode| following [=compute the key hash of reporting ID=].
-      1. Run [=update k-anonymity cache for key=] on |adReportingHashCode|.
+      1. If |igAd|'s [=interest group ad/selectable buyer and seller reporting IDs=] is not null:
+        1. [=list/For each=] |selectableReportingId| in |igAd|'s [=interest group ad/selectable buyer and seller reporting IDs=]:
+          1. Compute |adReportingHashCode| by getting the result of [=computing the key hash of reporting ID=] given |ig|, |igAd|, and |selectableReportingId|.
+          1. Run [=update k-anonymity cache for key=] on |adReportingHashCode|.
+      1. Otherwise:
+        1. Compute the |adReportingHashCode| following [=compute the key hash of reporting ID=].

```

Comment:
Same here re replacing Compute with Let. Also, this seems to be missing `given` arguments.